locations/spiders/mediamarkt.py

```diff
@@ -1,11 +1,22 @@
-from scrapy.linkextractors import LinkExtractor
-from scrapy.spiders import CrawlSpider, Rule
+import json
+
+from scrapy.http import Request
 
 from locations.structured_data_spider import StructuredDataSpider
 
 
-class MediaMarktSpider(CrawlSpider, StructuredDataSpider):
+class MediaMarktSpider(StructuredDataSpider):
     name = "media_markt"
     item_attributes = {"brand": "MediaMarkt", "brand_wikidata": "Q2381223"}
     start_urls = ["https://www.mediamarkt.de/de/store/store-finder"]
-    rules = [Rule(LinkExtractor(allow=r"https://www.mediamarkt.de/de/store/"), callback="parse_sd")]
+
+    def _parse(self, response):
```

Comment:
Perhaps a silly question, but what is the purpose of `_parse` versus the usual `parse`?

Reply:
Reference: https://github.com/scrapy/scrapy/blob/42b3a3a23b328741f1720953235a62cba120ae7b/scrapy/spiders/__init__.py#L78

I'm not sure I see what benefit is had from overwriting `_parse` instead of `parse` because when we want to parse structured data, `self.parse_sd` is being used as the callback and thus `StructuredDataSpider.parse` seems OK to overwrite?

Reply:
When I was testing it. It was not working with regular parse. But I just tried it again and it seems to work fine now. Not sure what the issue was when I first tested it.